examples/junit-spring-web/build-and-run-tests.sh

```diff
@@ -31,6 +31,19 @@ sed -i "/<artifactId>jazzer-junit<\/artifactId>/ {n;s/<version>.*<\/version>/<ve
 ./mvnw install:install-file -Dfile=../../bazel-bin/deploy/jazzer-project.jar       -DpomFile=../../bazel-bin/deploy/jazzer-pom.xml
 ./mvnw install:install-file -Dfile=../../bazel-bin/deploy/jazzer-api-project.jar   -DpomFile=../../bazel-bin/deploy/jazzer-api-pom.xml
 
+## Assert transitive JUnit dependencies are specified
+assertDependency() {
+  if ./mvnw dependency:tree | grep -q "$1"
+  then
+    echo "[SPRINGBOOT-JUNIT]: Found $1 dependency in project"
+  else
+    echo "[SPRINGBOOT-JUNIT]: Did not find $1 dependency in project"
+    exit 1
+  fi
+}
+assertDependency "org.junit.jupiter:junit-jupiter-api"
+assertDependency "org.junit.jupiter:junit-jupiter-params"
+assertDependency "org.junit.platform:junit-platform-launcher"
```

Comment:
This is good test coverage in the sense that it will prevent a regression of this fix, but I would be more comfortable with a test that actually verifies we can run the passing fuzz test with Jazzer without a crash.

This would require running Jazzer with the classpath obtained as done by cifuzz here: https://github.com/CodeIntelligenceTesting/cifuzz/blob/da75a4201cbd5587ca536162d48a60f37618b614/internal/build/maven/maven.go#L137-L169.

Reply:
Good point. I also added a test which directly invokes Jazzer and checks that no error is thrown. Did you have something like this in mind?